### PR TITLE
send callback arg ptrs from c to dart as Uint8List

### PR DESCRIPTION
**Pull request by unsuitable001**

### CODE REVIEW

File: lib/src/http_callback_handler.dart [hunk 1]
```diff
@@ -98,8 +98,7 @@ class CallbackHandler {
     receivePort.listen((dynamic message) {
       final reqMessage =
           _CallbackRequestMessage.fromCppMessage(message as List);
-      Int64List args;
-      args = reqMessage.data.buffer.asInt64List();
+      Uint64List args = reqMessage.data.buffer.asUint64List();
```
Comment (dcharkes) [inline lib/src/http_callback_handler.dart:101]: final args =

File: src/wrapper.cc [hunk 2]
```diff
@@ -124,16 +124,16 @@ Dart_CObject CallbackArgBuilder(int num, ...) {
   Dart_CObject c_request_data;
   va_list valist;
   va_start(valist, num);
-  void *request_buffer = malloc(sizeof(uint64_t) * num);
-  uint64_t *buf = reinterpret_cast<uint64_t *>(request_buffer);
+  void *request_buffer = malloc(sizeof(intptr_t) * num);
+  intptr_t *buf = reinterpret_cast<intptr_t *>(request_buffer);
```
Comment (dcharkes) [inline src/wrapper.cc:128]: This should be kelt uint64_t if you're using it only with asUint64List in Dart.

We can swap everything to be word-size (4 byte words or 8 byte words) specific when we start supporting 32 bit.

**Review by dcharkes — APPROVED** [review]
Please bump the version number and add a thing in the changelog.

Add a note in the changelog to https://dart-review.googlesource.com/c/sdk/+/207865 that changed the behavior of Dart_PostCObject with Dart_TypedData_kUint64.

File: src/wrapper.cc (hunk 2 above)
Comment (unsuitable001) [reply src/wrapper.cc:128]: Okay :)

**Pull request merged**